### PR TITLE
Fix bug 57688 : [DotNet] User Id doesn't work for SDK 1.13.2

### PR DESCRIPTION
**Pull request by JasoWu**
Things to consider before you submit the PR:
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

We found that press enter after input any key the key action will be 'DOWN', Otherwise it will be 'UP',
So add another choice to set the user id.
image

### CODE REVIEW

File: Apps/Contoso.Android.Puppet/ModulePages/AppCenterFragment.cs [hunk 1]
```diff
@@ -114,7 +114,7 @@ private void LogWriteLevelClicked(object sender, EventArgs e)
 
         private void UserIdTextKeyPressedHandler(object sender, View.KeyEventArgs e)
         {
-            if (e.Event.Action == KeyEventActions.Down && e.KeyCode == Keycode.Enter)
+            if ((e.Event.Action == KeyEventActions.Down && e.KeyCode == Keycode.Enter) || (e.Event.Action == KeyEventActions.Up))
```
Comment (MatkovIvan) [inline Apps/Contoso.Android.Puppet/ModulePages/AppCenterFragment.cs]: The first condition became redundant now

Comment (JasoWu) [reply Apps/Contoso.Android.Puppet/ModulePages/AppCenterFragment.cs]: Yeah, You're right, Thank you!

**Comment by JasoWu** [comment]
> What about all the other applications? I don't see an update on demo app while the bug was reported against the demo. You modified only 1 of the puppets.

**Comment by JasoWu** [comment]
@guperrot I have already found the problems for other platform applications, And will send PR today. This PR is just to get your suggestions.

**Pull request merged**